web/src/components/modals/delete-pep.tsx

```diff
@@ -75,12 +59,10 @@ export const DeletePEPModal: FC<Props> = ({ show, onHide, namespace, project, ta
       </Modal.Body>
       <Modal.Footer>
         <button
-          onClick={() => mutation.mutate()}
-          disabled={confirmText !== `${namespace}/${project}:${tag}` || mutation.isLoading}
-          type="button"
-          className="btn btn-danger"
-        >
-          {mutation.isLoading ? 'Deleting...' : 'Yes, delete'}
+          onClick={deleteProjectMutation}
+          disabled={isLoading}
+          type="button">
```

Comment:
yep, fixed this

Reply:
Ok, I don't know why but I still had to pass isLoading in order for it to work, otherwise it says it didn't exist iirc

https://github.com/pepkit/pephub/commit/b62bb8de755f91ef4bfce011bfb6bdefd9e1a56c#diff-96719a3b1fa5da1bc6ebb0b72f05e19137eceef467248b7556cb062be42ddad2